sirepo/feature_config.py

```diff
@@ -54,7 +55,7 @@ def auth_controlled_sim_types():
       frozenset:  enabled sim types that require role
     """
     return frozenset(
-        cfg().proprietary_sim_types.union(cfg().default_proprietary_sim_types),
+        cfg().proprietary_sim_types.union(cfg().proprietary_with_moderation_prompt_sim_types),
```

Comment:
I think this is `moderated_sim_types`. They may or may not be proprietary.

Reply:
84a51a4c4d292b403b922c73411c9bad58de76bf